Automattic/wp-calypso
# Signup: remove unused steps (Take 2)

Related to #

## Proposed Changes

* #91873 had to be reverted because the `emails` step was removed from the step definitions, but it was still used when calling the `maybeExcludeEmailsStep` function. 
* The `maybeExcludeEmailsStep` has only one usage, so it can be safely removed.

## Why are these changes being made?


* Please see #91873.

## Testing Instructions



* Since #91873 was already reviewed and tested, these are the instructions for the new changes:
* Code changes are in https://github.com/Automattic/wp-calypso/pull/92058/commits/9e1dade9703f0bb44ed061791f78e36094c62f9a
* Go to `/start/with-theme/domains-theme-preselected?theme=tronar&intervalType=yearly` and confirm that you can complete the flow.

## Pre-merge Checklist
